.dotnet/src/Custom/OpenAIClient.cs

```diff
@@ -28,12 +28,16 @@ namespace OpenAI;
 [CodeGenSuppress("GetLegacyCompletionClientClient")]
 public partial class OpenAIClient
 {
+    protected readonly ApiKeyCredential _cachedCredential = null;
+    protected readonly OpenAIClientOptions _cachedOptions = null;
+
+    protected internal static RequestOptions DefaultRequestOptions = new();
+    protected internal static PipelineMessageClassifier PipelineMessageClassifier200
```

Comment:
Is there a reason to make this protected instead of defining it internally on the Azure package side?  It is super easy to create an instance of and I can't think of a reason there's value in inheriting it from the base client if it's lazily intantiated?

Reply:
If it's not protected, won't we end up with an extra, needless copy? Not that it's huge either way, of course--if it's preferrable to keep the encapsulation as simple as possible even at the expense of the extra copy, that seems totally reasonable.

Reply:
Wait, I think I see -- if make it a lazy field pair in each case, this isn't a copy issue anyway. I just oversimplified by making a direct-init property. I'll restore the old pattern!